init-scripts/abrt.conf

```diff
@@ -1,3 +1,9 @@
-#Type Path          Mode UID  GID
-d     /var/tmp/abrt 0755 abrt abrt
+#Type Path                  Mode UID  GID
+d     /var/tmp/abrt         0755 abrt abrt
 x     /var/tmp/abrt/*
+
+d     /run/abrt             0755 root root
+x     /run/abrt/abrt.pid
+x     /run/abrt/abrt.socket
```

Comment:
I would rather use 

    r!     /run/abrt/abrt.pid
    r!     /run/abrt/abrt.socket

Reply:
Yeah, that makes more sense.

Reply:
Changed.